examples/miral-shell/shell_main.cpp

```diff
@@ -334,10 +336,60 @@ int main(int argc, char const* argv[])
             return false;
     };
 
+    // The following filter triggers the application selector internal application
+    // on "Ctrl + Tab" and "Ctrl + Shift + Tab". Note that this does NOT use
+    // "Alt + Tab" as that is claimed by the MinimalWindowManager for the time being.
```

Comment:
I don't think we want to duplicate this into miral-shell as well as mir_demo_server - in its current state it is not an example we want anyone to adopt.